Specs/Number/Arabic/NumberModel.json

```diff
@@ -2001,9 +1980,9 @@
     ]
   },
   {
-    "Input": "عدد الطلاب في الدولة ١،٢٣٤،٥٦٧",
+    "Input": "الأرقام هي ١ ، ٢٣٤ ، ٥٦٧",
```

Comment:
Same, and so on.

Reply:
Added brief in PR Description.

Reply:
I'll merge, but can you add more info there about the types of issues that led to the corrections? Wrong grammar, not exactly the same meaning, invalid sentences? This is very valuable feedback to track. Thanks!